src/components/Project.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+const Project = (props) => {
+      return (
```

Comment:
To prevent longer form `props.thing.thing` references, it would be more clean to destructure whatever you need from props like `const { name, display_image, name, description, hosted_url } = props.project`